Formula/sdees.rb

```diff
@@ -0,0 +1,15 @@
+class Sdees < Formula
+  desc "Serverless Decentralized Editing of Encrypted Stuff"
+  homepage "https://github.com/schollz/sdees"
+  url "https://github.com/schollz/sdees/releases/download/1.2.1/sdees_osx_amd64.zip"
```

Comment:
Sorry for that, tried to figure out if build or source release is needed. Will update this PR.